### PR TITLE
`docs/features`: Update security features page

### PR DESCRIPTION
**Pull request by razvand**
Update security features page to the current state of things.

### CODE REVIEW

**Review by michpappas — APPROVED** [review]
Looks good 🚀 

Reviewed-by: Michalis Pappas <michalis@unikraft.io>

File: content/en/docs/features/security.md [hunk 1]
```diff
@@ -49,10 +50,12 @@ This effort is by nature a continuous, community-driven task, but we are expecti
 | [ARM Branch Target Identification (BTI)](https://github.com/unikraft/unikraft/pull/421)                | Upstream         | `ARCH_ARM_64`                  |
 | [ARM Memory Tagging Extension (MTE)](https://github.com/unikraft/unikraft/pull/458)                    | Upstream         | `ARCH_ARM_64`                  |
 | [ARM True Random Number Generator (RNG)](https://github.com/unikraft/unikraft/pull/434)                | Upstream         | `ARCH_ARM_64`                  |
+| [x86_64 True Random Number Generator (RNG)](https://github.com/unikraft/unikraft/pull/695)             | Upstream         | `ARCH_X86_64`                  |
 | [Kernel Address Sanitizer (KASAN)](https://github.com/unikraft/unikraft/pull/191)                      | Under review     | `PLAT_KVM && ARCH_X86_64`      |
 | [Position Independent Executables (PIE)](https://github.com/unikraft/unikraft/pull/239)                | Under review     | `PLAT_KVM && ARCH_X86_64`      |
```
Comment (StefanJum) [inline content/en/docs/features/security.md:55]: Also change this link to point to https://github.com/unikraft/unikraft/pull/772

**Review by StefanJum — CHANGES_REQUESTED** [review]
See the comments @razvand

**Review by mariasfiraiala — APPROVED** [review]
Besides @StefanJum's comments, this looks alright. Thanks @razvand!

Reviewed-by: Maria Sfiraiala <maria.sfiraiala@gmail.com>

File: content/en/docs/features/security.md [hunk 2]
```diff
@@ -49,10 +50,12 @@ This effort is by nature a continuous, community-driven task, but we are expecti
...
-| [x86_64 True Random Number Generator](https://github.com/unikraft/unikraft/pull/420)                   | Under review     | `ARCH_X86_64`                  |
-| [Shadow Stack](https://github.com/unikraft/unikraft/pull/505)                                          | In Progress      | `ARCH_ARM_64`                  |
+| [Shadow Stack](https://github.com/unikraft/unikraft/pull/505)                                          | In progress      | `ARCH_ARM_64`                  |
+| Arm CCA (Confidential Compute Architecture)                                                            | In progress      | `ARCH_ARM_64` (Armv9)          |
```
[the remark this answers is not in the document]
Comment (michpappas) [reply content/en/docs/features/security.md:57]: A bit too draft to put on the front page IMO. It's just there to collect changes, but there will be individual PRs shortly.

[the remark this answers is not in the document]
Comment (StefanJum) [reply content/en/docs/features/security.md:57]: Ok, makes sense.

**Comment by StefanJum** [comment]
Covered by the website refactoring.

**Pull request closed without merging**